kong/plugins/zipkin/reporter.lua

```diff
@@ -25,6 +25,26 @@ local span_kind_map = {
 	producer = "PRODUCER";
 	consumer = "CONSUMER";
 }
+
+-- Utility function to convert opentracing tags to zipkin tags
+-- values of zipkin tags should be strings, see [zipkin-api](https://zipkin.io/zipkin-api).
+function to_zipkin_tags(opentracing_tags)
+	zipkin_tags = {}
+	if not opentracing_tags then
+		return zipkin_tags
+	end
+
+	for k, v in pairs(opentracing_tags) do
```

Comment:
Use [`span:each_tag()`](https://github.com/Kong/opentracing-lua/blob/ee1cdc12b32097f6b9b4379e3ce207ecfaab2988/opentracing/span.lua#L89) here (anything else is relying on undefined internals)